### PR TITLE
Observability maturity model

### PR DESCRIPTION
**Pull request by kiranprakash-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/en/guides/operational/observability-maturity-model.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Analyze these alerts on a regular basis and automate remediation for common repeated alerts. Share and communicate the alert findings with relevant teams to provide feedback on operational and process improvement.
+
+Develop a plan to gradually build a knowledge graph that helps you correlate different entities and understand the dependencies between different parts of a system. It enables customers visualize the impact of changes to a system, helping to predict and mitigate potential issues.
+
+### Stage 3: Advanced Observability - Traces for Correlation and Anomaly Detection
```
Comment (toshaldudhwala) [inline docs/en/guides/operational/observability-maturity-model.md:49]: please remove "traces"

File: docs/en/images/o11y4AIOps.png
Comment (toshaldudhwala) [inline docs/en/images/o11y4AIOps.png:1]: replace x-ray logo

File: docs/en/images/AWS_O11y_Stack.png
Comment (toshaldudhwala) [inline docs/en/images/AWS_O11y_Stack.png:1]: replace x-ray logo

File: docs/en/guides/operational/observability-maturity-model.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+The assessment helps identify the targeted metrics and KPIs that lays the foundation for further development and optimization of the existing layout. The assessment of your Observability Maturity Model plays a crucial role in ensuring that your business is prepared to handle the complex, dynamic nature of modern systems. It aids in identifying blind spots and areas of weakness that could potentially lead to system failures or performance issues.
+
+Moreover, regular assessments ensure that your business remains agile and adaptable. It allows you to keep pace with evolving technologies and methodologies, thereby ensuring that your systems are always at the peak of efficiency and reliability.
+To have an assessment done for the workloads in your organization, please contact your AWS account team for further information and assistance.
```
Comment (toshaldudhwala) [inline docs/en/guides/operational/observability-maturity-model.md:90]: please remove this line

Comment (kiranprakash-aws) [reply docs/en/guides/operational/observability-maturity-model.md:90]: Done.

File: docs/en/guides/operational/observability-maturity-model.md (hunk 1 above)
Comment (kiranprakash-aws) [reply docs/en/guides/operational/observability-maturity-model.md:49]: updated the title

**Review by lewinkedrs — APPROVED** [review]
LGTM

**Review by lewinkedrs — COMMENTED** [review]
Hey why is mkdocs.yaml deleted here. Can you fix this so we can merge?

**Review by lewinkedrs — APPROVED** [review]
looks good

**Pull request merged**